libcxx/include/__ostream/basic_ostream.h

```diff
@@ -353,9 +354,9 @@ _LIBCPP_HIDE_FROM_ABI basic_ostream<_CharT, _Traits>& operator<<(basic_ostream<_
       typedef ostreambuf_iterator<_CharT, _Traits> _Ip;
       if (std::__pad_and_output(
               _Ip(__os),
-              &__c,
-              (__os.flags() & ios_base::adjustfield) == ios_base::left ? &__c + 1 : &__c,
-              &__c + 1,
+              std::addressof(__c),
+              std::addressof(__c) + ((__os.flags() & ios_base::adjustfield) == ios_base::left),
```

Comment:
I'm not a big fan of using a `bool` as an `int` here. I'd even be OK with `addressof(c) + (bool-condition ? 1 : 0)`.